docs/examples/warp_3D.ipynb

```diff
@@ -0,0 +1,381 @@
+    "$${Out}(x, y, z) = {In}(x_{src}, y_{src}, z_{src})$$\n",
+    "\n",
+    "This way each output pixel is calculated exactly once.\n",
+    "\n",
+    "If the source coordinates do not point exactly to pixel centers, the values of neighboring pixels will be interpolated or the nearest pixel is taken, depending on the interpolation method specified in the `interp_type` argument.\n",
+    "\n",
+    "### Affine transform\n",
+    "\n",
+    "The source sample coordinates $x_{src}, y_{src}, z_{src}$ are calculated according to the formula:\n",
+    "\n",
+    "$$\n",
+    "\\begin{vmatrix}\n",
+    "x_{src} \\\\\n",
+    "y_{src} \\\\\n",
+    "z_{src}\n",
+    "\\end{vmatrix}\n",
+    "= \\begin{vmatrix}\n",
+    "m_{00} & m_{01} & m_{02} & t_x \\\\\n",
+    "m_{10} & m_{11} & m_{12} & t_y \\\\\n",
+    "m_{20} & m_{21} & m_{22} & t_z\n",
+    "\\end{vmatrix}\n",
+    "\\begin{vmatrix}\n",
+    "{x} \\\\\n",
+    "{y} \\\\\n",
+    "{z} \\\\\n",
+    "1\n",
+    "\\end{vmatrix}\n",
+    "$$\n",
+    "\n",
+    "Where $x, y$ are coordinates of the destination pixel and the matrix represents the inverse (destination to source) affine transform. The \n",
+    "$\\begin{vmatrix}\n",
+    "m_{00} & m_{01} & m_{02} \\\\\n",
+    "m_{10} & m_{11} & m_{12} \\\\\n",
+    "m_{20} & m_{21} & m_{22}\n",
+    "\\end{vmatrix}$ block represents a combined rotate/scale/shear transform and $t_x, t_y, t_z$ is a translation vector.\n",
+    "\n",
+    "### 3D Rotation\n",
+    "\n",
+    "Rotate operator is implemented in terms of affine transform, but calculates the transform matrix internally. The output size is automatically adjusted and the size parity is adjusted to reduce blur near the volume centre. The rotation is defined by specifying axis (as a vector) and angle (in degrees).\n",
+    "\n",
+    "The rotation matrix (source-to-destination) is calculated as:\n",
+    "\n",
+    "$$\n",
+    "\\begin{vmatrix}\n",
+    "u^2 + (v^2 + w^2) \\cdot \\cos \\alpha &\n",
+    "uv\\cdot(1-\\cos \\alpha) - w\\cdot \\sin \\alpha &\n",
+    "uw\\cdot(1-\\cos \\alpha) + v\\cdot \\sin \\alpha \\\\\n",
+    "uv\\cdot(1-\\cos \\alpha) - w\\cdot \\sin \\alpha &\n",
+    "v^2 + (u^2 + w^2) \\cdot \\cos \\alpha &\n",
+    "vw\\cdot(1-\\cos \\alpha) - u\\cdot \\sin \\alpha \\\\\n",
+    "uw\\cdot(1-\\cos \\alpha) - v\\cdot \\sin \\alpha &\n",
+    "vw\\cdot(1-\\cos \\alpha + u\\cdot \\sin \\alpha &\n",
+    "w^2 + (u^2 + w^2) \\cdot \\cos \\alpha\n",
+    "\\end{vmatrix}\n",
+    "$$\n",
+    "\n",
+    "where $u, v, w$ is normalized `axis` vector and $\\alpha$ is `angle` converted from degrees to radians.\n",
+    "\n",
+    "Note that internally, the inverse matrix is used to achieve destination-to-source mapping, which is how the operation is actually implemented."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Usage example\n",
+    "\n",
+    "First, let's import the necessary modules and define the location of the dataset."
```

Comment:
`and define the location of the dataset.` I think is it not done here

Reply:
True.